django/utils/archive.py

```diff
@@ -114,8 +114,7 @@ def _copy_permissions(mode, filename):
             os.chmod(filename, mode)
 
     def split_leading_dir(self, path):
-        path = str(path)
-        path = path.lstrip("/").lstrip("\\")
+        path = str(path).lstrip("/").lstrip("\\")
```

Comment:
Thanks for this patch :+1:, however we try to avoid trivial code reformatting unless it's a big problem as it creates extra noise in the history.